code/game/objects/items/weapons/cosmetics.dm

```diff
@@ -133,7 +133,7 @@
 			if((C.species && C.species.flags & ALL_RPARTS) && robohead.is_monitor) //If the target is of a species that can have prosthetic heads, but the head doesn't support human hair 'wigs'...
 				to_chat(user, "<span class='warning'>You find yourself disappointed at the appalling lack of hair.</span>")
 				return
-			if(C.h_style == "Bald" || C.h_style == "Balding Hair" || C.h_style == "Skinhead")
+			if(C.h_style == "Bald" || C.h_style == "Balding Hair" || C.h_style == "Skinhead" || findtextEx(C.h_style, "Skrell"))

```

Comment:
It would probably be better to just add another check to this like the above
```
if(C.get_species() == "Skrell")
    to_chat(user, "<span class='warning'>Your razor isn't going to cut through tentacles.</span>")
    return
```